src/features/cards/organisms/skeleton-list.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import * as React from "react"
+import { CardsList } from "./cards-list"
+import { CardSkeleton } from "./card-skeleton"
+
+type Props = {
+  isLoading: boolean,
+  ids: number[],
+  count?: number,
+  renderEmpty: () => React.Node,
+  renderCard?: *,
+}
+
+export const SkeletonList = ({
+  isLoading,
+  ids,
+  count = 3,
+  renderEmpty,
+  renderCard,
+}: Props) =>
+  isLoading ? (
+    <>
+      {Array.from({ length: count }, (_, idx) => (
+        <CardSkeleton key={idx} />
+      ))}
+    </>
+  ) : (
+    <CardsList ids={ids} renderEmpty={renderEmpty} renderCard={renderCard} />
+  )
+
+SkeletonList.defaultProps = {
+  count: undefined,
```

Comment:
It doit nothing, why it needs?

Reply:
Because eslint can't see default parameter